pkg/ebpf/c/tracee.bpf.c

```diff
@@ -657,6 +657,7 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
 
 // number of iterations - value that the verifier was seen to cope with - the higher, the better
 #define MAX_NUM_MODULES 450
+#define MAX_NUM_LOOPS_MODULES 100000
```

Comment:
Its a bit much, no?
Why not 10k or even less?

Reply:
It stops when finishes - so I want a value that if it happens, it'll certainly indicate a problem. 10k is also probably ok but doesn't make much different than 100k.